platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/paos/PaosInInterceptor.java

```diff
@@ -243,6 +246,18 @@ public void handleMessage(Message message) throws Fault {
       if (httpResponse.statusCode < 400) {
         httpResponseContent = httpResponse.content;
         message.setContent(InputStream.class, httpResponseContent);
+        Map<String, List<String>> headers = (Map) message.get(Message.PROTOCOL_HEADERS);
+        if (headers != null) {
```

Comment:
not very familiar with this code but If the headers are null should we just create a new map and populate that?

Reply:
yes it absolutely should. i'm going to blame this on vacation brain . Fixed 👍